### DISCUSSION TITLE
I just want to thank the people that develop this program.

### DISCUSSION THREAD

**Discussion by ThGrSoRu** [sxyazi/yazi#260] [1. Q&A]
Out of many other terminal file managers, this one struck me with its features and integrations, while not being overwhelming to configure and customize.
All while completely written in Rust!
I'll probably daily drive this forever.
Huge thanks <3

**Comment by sxyazi** [comment]
It means a lot to me to hear about your positive experience.

Thank you for taking the time to share your thoughts, and let me know that my work has a meaningful impact on you. ❤️
